packages/camera/src/components/Controls/index.js

```diff
@@ -113,7 +114,7 @@ export default function Controls({
         || hasNoIdle || isAddDamageButtonAndDisabled(id) ? styles.buttonDisabled : {},
       ],
       disabled: rest.disabled || loading || hasNoIdle || isAddDamageButtonAndDisabled(id),
-    }, children)), [
+    }, cloneElement(children, {isPortraitModeVinLayoutView}))), [
```

Comment:
Can we add transform - rotate on parent component instead of setting on each & every button individually?